examples/data_augmentation/uda/utils/model_utils.py

```diff
@@ -0,0 +1,42 @@
+"""
+Model utility functions
+"""
+import math
+import torch
+
+
+def get_lr_multiplier(step: int, total_steps: int, warmup_steps: int) -> float:
```

Comment:
These docstrings do not follow our standard. Please document every parameter, an example:

https://github.com/asyml/forte/blob/master/forte/utils/utils_io.py#L28